assets/js/components/HostsList.jsx

```diff
@@ -23,6 +24,27 @@ const getHeartbeatIcon = ({ heartbeat }) => {
   }
 };
 
+const getSAPInstanceFromSID = (sid, sapSystems) => {
+  const instance = sapSystems.find(
+    (instance) => instance.instance?.sid === sid
+  );
+
+  return instance;
+};
+
+const getSapSystemsByHost = (sapSystems, hostId) => {
```

Comment:
I believe this can be a selector on the state, selecting all the instances on a host.

Reply:
Ah! nice, I see you used it in HostDetails, going to give it a try

Reply:
I'll create a ticket to figure out later how to use this. It seems to work on first load, but breaks after a refresh. I suspect that `useSelector` is not intended to be used each time we iterate on the hosts list to display each entry of the table. The warning itself after it breaks is:
>Warning: React has detected a change in the order of Hooks called by HostsList. This will lead to bugs and errors if not fixed. For more information, read the Rules of Hooks: https://reactjs.org/link/rules-of-hooks